.mise.toml

```diff
@@ -0,0 +1,9 @@
+[tools]
+node = '20.10.0'
+# stuck on this version of yarn because of some weird problems
+# related to babel tooling when installing new dependencies
+yarn = '1.19.0'
+
+[env]
+RTX_FETCH_REMOTE_VERSIONS_TIMEOUT="300 s"
```

Comment:
is there a mise alternative for this variable?

Reply:
I think so, but it's not documented very well. I updated -- thanks for pointing that out.

(This timeout is needed because node.org can have excessively slow download times. Not sure if that's fixed at this point.)